float32 yaw_offset